tests/e2e/spec/tonedear.e2e.spec.js

```diff
@@ -0,0 +1,176 @@
+        describe('Tonedear Test Suite ' + (mode === 'jquery' ? '[JQuery mode]' : '[SW mode]'), function () {
+            this.timeout(60000);
+            this.slow(10000);
+
+            it('Load Kiwix JS and verify title', async function () {
+                await driver.get(`http://localhost:${port}/dist/www/index.html?noPrompts=true`);
+                await driver.sleep(1300);
+                await driver.navigate().refresh();
+                await driver.sleep(800);
+                const title = await driver.getTitle();
+                assert.equal('Kiwix', title);
+            });
+
+            it('Switch to ' + mode + ' mode', async function () {
+                const modeSelector = await driver.wait(
+                    until.elementLocated(By.id(mode + 'ModeRadio'))
+                );
+                await driver.executeScript(
+                    'var el=arguments[0]; el.scrollIntoView(true); setTimeout(function() {el.click();}, 50); return el.offsetParent;',
+                    modeSelector
+                );
+                await driver.sleep(1300);
+
+                try {
+                    const activeAlertModal = await driver.findElement(
+                        By.css('.modal[style*="display: block"]')
+                    );
+                    if (activeAlertModal) {
+                        serviceWorkerAPI = await driver.findElement(By.id('modalLabel'))
+                            .getText()
+                            .then(function (alertText) {
+                                return !/ServiceWorker\sAPI\snot\savailable/i.test(alertText);
+                            });
+                        const approveButton = await driver.wait(
+                            until.elementLocated(By.id('approveConfirm'))
+                        );
+                        await approveButton.click();
+                    }
+                } catch (e) {
+                    // Do nothing
+                }
+            });
+
+            it('Load Tonedear archive and verify content', async function () {
+                if (!serviceWorkerAPI) {
+                    console.log('\x1b[33m%s\x1b[0m', '    - Following test skipped:');
+                    this.skip();
+                }
+
+                const archiveFiles = await driver.findElement(By.id('archiveFiles'));
+                await driver.executeScript('arguments[0].style.display = "block";', archiveFiles);
+
+                if (!BROWSERSTACK) {
+                    await archiveFiles.sendKeys(tonedearBaseFile);
+                    await driver.executeScript('window.setLocalArchiveFromFileSelect();');
+                } else {
+                    await driver.executeScript(
+                        'window.setRemoteArchives.apply(this, [arguments[0]]);',
+                        [tonedearBaseFile]
+                    );
+                    await driver.sleep(1300);
+                }
+            });
+
```

Comment:
In between these two tests, there should be another test that switches to the page "Android & iOS App", and verifies that the correct page has loaded. Then it will be possible to test the dimension of the two images on that page in the test below. See screenshot.

![image](https://github.com/user-attachments/assets/03e2f205-8886-4798-a1c5-d60528f870eb)

Reply:
Got it sir! I didn't thought this was causing the issue😅